raiden/constants.py

```diff
@@ -31,16 +32,24 @@ class EthClient(Enum):
 START_QUERY_BLOCK_KEY = 'DefaultStartBlock'
 SNAPSHOT_STATE_CHANGES_COUNT = 500
 
+GAS_LIMIT_UPPER_BOUND = int(0.4 * 3_141_592)
```

Comment:
This constant and its intention should be explained somewhere -- its naming is very general and I cannot make sense of it.

Reply:
This is the upper bound for a transaction gas limit, would any of these be better?

- `TRANSACTION_GAS_LIMIT_UPPER_BOUND`
- `TRANSACTION_GAS_LIMIT_MAXIMUM`